client/src/modules/templates/modals/report.save.html

```diff
@@ -6,32 +6,38 @@
 </div>
 
 <form name="SaveForm" bh-submit="SaveCtrl.submit(SaveForm)" autocomplete="off" novalidate>
-<div class="modal-body">
-  <div class="form-group"
-    ng-class="{ 'has-error' : SaveForm.$submitted && SaveForm.label.$invalid }">
-    <label class="control-label" translate>FORM.LABELS.LABEL</label>
-    <input class="form-control" id="label" name="label" autocomplete="off" ng-model="SaveCtrl.documentOptions.label" required />
+  <div class="modal-body">
+    <div class="form-group" ng-class="{ 'has-error' : SaveForm.$submitted && SaveForm.label.$invalid }">
+      <label class="control-label" translate>FORM.LABELS.LABEL</label>
+      <input class="form-control" id="label" name="label" autocomplete="off" ng-model="SaveCtrl.documentOptions.label" required
+      />
 
-    <div class="help-block" ng-messages="SaveForm.label.$error" ng-show="SaveForm.$submitted">
-      <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+      <div class="help-block" ng-messages="SaveForm.label.$error" ng-show="SaveForm.$submitted">
+        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+      </div>
     </div>
-  </div>
 
-  <div class="form-group">
-    <label class="control-label" translate>REPORT.UTIL.SAVE_AS</label>
-    <select class="form-control" ng-model="SaveCtrl.documentOptions.renderer" ng-options="type.key as (type.label | translate) for type in SaveCtrl.supportedRenderTypes">
-    </select>
+    <div class="form-group">
+      <label class="control-label" translate>REPORT.UTIL.SAVE_AS</label>
+      <select class="form-control" ng-model="SaveCtrl.documentOptions.renderer" ng-options="type.key as (type.label | translate) for type in SaveCtrl.supportedRenderTypes">
+      </select>
+    </div>
+    <!-- layout orientation -->
+    <bh-render-option
+      on-change="SaveCtrl.setDocumentorientation"
```

Comment:
If this is a function, it should be called in the HTML.  I'm not sure why AngularJS does this, but it's how all callbacks seem to work with AngularJS.  Maybe it is for clarity.

The best way to structure this is:
```html
<bh-render-option
  on-change="SaveCtrl.setDocumentOrientation(orientation)">
</bh-render-option>
```

... then your JS in the component can be:
```js
$ctrl.setOrientation = () => $ctrl.onChange({ orientation : $ctrl.orientation });
```
And I think that should work.